api/types/settings/platform.go

```diff
@@ -230,19 +230,23 @@ func (ps *Platform) SetUpdatedBy(v string) {
 
 // Update takes another settings record and updates the internal fields, intended
 // to be used when the refreshing settings record shared across the server.
-func (ps *Platform) Update(newSettingps *Platform) {
+func (ps *Platform) Update(_ps *Platform) {
 	if ps == nil {
 		return
 	}
 
-	if newSettingps == nil {
+	if _ps == nil {
 		return
 	}
 
-	ps.SetCompiler(newSettingps.GetCompiler())
-	ps.SetQueue(newSettingps.GetQueue())
-	ps.SetRepoAllowlist(newSettingps.GetRepoAllowlist())
-	ps.SetScheduleAllowlist(newSettingps.GetScheduleAllowlist())
+	ps.SetCompiler(_ps.GetCompiler())
+	ps.SetQueue(_ps.GetQueue())
+	ps.SetRepoAllowlist(_ps.GetRepoAllowlist())
+	ps.SetScheduleAllowlist(_ps.GetScheduleAllowlist())
+
+	ps.SetCreatedAt(_ps.GetCreatedAt())
```

Comment:
`created_at` shouldn't be updateable, right?

Reply:
no but that raises a good point this function should really be called `DuplicateInPlace` or something like that. i see the confusion

Reply:
or... `Overwrite` or `FromSettings` ?

fwiw its used in these places:
https://github.com/go-vela/server/blob/main/cmd/vela-server/server.go#L255
https://github.com/go-vela/server/blob/main/api/admin/settings.go#L121-L122

Reply:
i made some tweaks and renamed `Update` to `FromSettings`